### PR TITLE
README: Improvements based on some FAQ

### PR DESCRIPTION
**Pull request by martomi**
- Apply formatting to both WINDOWS.md and README.md
- Add link from README to new Status Reference wiki page
- Add clarifications about multi-harvester setup monitoring
- Add link to disappearing harvester issue on WINDOWS (log rotation)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ on [Remote Monitoring Multiple Harvesters](https://github.com/martomi/chiadog/wi
 You can enable more verbose logging from `config.yaml` by changing `INFO` to `DEBUG`. You should see logs for every
 keep-alive event from the harvester.
 
+You can also set the `debug_mode` to `true` in `config.yaml` under `daily_stats`. In this mode `chiadog` will send
```
Comment (pieterhelsen) [inline README.md:169]: I would contain this to PR #176 
Also, if you agree to my suggestion, this would be omitted in favor of the argparser approach

**Review by pieterhelsen — CHANGES_REQUESTED** [review]
Everything looks good! 👍  Only the changes relating `debug_mode` should be added to the proper PR in my opinion.

File: README.md (hunk 1 above)
Comment (martomi) [reply README.md:169]: 👍 Will drop that change from README

**Review by pieterhelsen — APPROVED** [review]
Ready to go!

**Pull request merged**